### ISSUE TITLE
Enhancement: Active Directory integration

### ISSUE THREAD

**Issue by red-erik** [projectsend/projectsend#853]
Hello,
I saw some issues related to Ad or LDAP (probably I opened one of those) but I never find a complete guide or solution to implement AD auth for client.
Don't you think it would be a great enhancement to the product?

Regards,

Red.

**Comment by ignacionelson** [comment]
Hi! Thanks for your comment!
I've done some work already to implement LDAP authentication but the code is commented out in the release since it's incomplete.
There's not too much missing, but I'm far from an expert in LDAP and I've had issues setting up a standard set of options with the necessary fields to accommodate to all possible connection cases. I need some advice from real LDAP users to finish the implementation.

**Comment by red-erik** [comment]
Hello,
I can help you in testing the solution and share some LDAP knowledge with you.
Anyway, there is a lot of "ready to use" stuff around. Pls, have a look at (for example)
https://github.com/DirectoryTree/LdapRecord
Let me know...

Red.

**Comment by kkplein** [comment]
Yes, we're also interested in LDAP/AD support, and since we are projectsend users...: testing it (or sharing what limited knowledge we have) would be a nice way to contribute back.

**Comment by red-erik** [comment]
Hello,
any news on this topic ?
regards,

Red.

**Comment by mikkeljohnsen** [comment]
Any news on this ?

This would be great if it was finished

**Comment by ignacionelson** [comment]
Implemented in r1945
